src/context/playlists-context.js

```diff
@@ -35,16 +58,64 @@ const PlaylistsProvider = ({ children }) => {
           type: "MANAGE_PLAYLIST",
           payload: playlists,
         });
+        const playlistId = playlists[playlists.length - 1]._id;
+        addVideoToPlaylist(playlistId, video);
       }
       console.log(playlists);
     } catch (err) {
       console.log(err);
     }
   };
 
+  const deletePlaylist = async (playlistId) => {
+    try {
+      const response = await axios.delete(`${PLAYLISTS_API}/${playlistId}`, {
+        headers: { authorization: encodedToken },
+      });
+
+      const { playlists } = response.data;
+      if (response.status === 200) {
+        playlistsDispatcher({
+          type: "MANAGE_PLAYLIST",
+          payload: playlists,
+        });
```

Comment:
- the way you created variable for apis, you can also create variable for dispatch type as well instead of using string "MANAGE_PLAYLIST"

Reply:
Thanks for suggesting will look into it.